### PR TITLE
Add 6 v2v function cases

### PR DESCRIPTION
**Pull request by chloerh**
2 with spice type
1 with wrong fstab file
3 with selinux configuration

### CODE REVIEW

File: v2v/tests/src/specific_kvm.py [hunk 1]
```diff
@@ -576,75 +609,90 @@ def check_result(result, status_error):
...
+        elif checkpoint == 'host_selinux_on':
+            params['selinux_stat'] = utils_selinux.get_status()
+            utils_selinux.set_status('enforcing')
+        elif checkpoint.startswith('selinux'):
+            if set_selinux(checkpoint[8:]):
+                vm = libvirt_vm.VM(vm_name, params, test.bindir,
+                                   env.get('address_cache'))
+                # Reboot vm to make configuration effect
+                vm.reboot()
+                vm.shutdown()
```
Comment (will-Do) [inline v2v/tests/src/specific_kvm.py]: Why not shutdown directly, and next time why VM start, your configuration will effect, right?

File: v2v/tests/src/specific_kvm.py [hunk 2]
```diff
@@ -450,6 +452,22 @@ def cleanup_fs():
...
+        session = kwargs['session']
+        current_stat = session.cmd_output('getenforce').strip()
+        logging.debug('Current selinux status: %s', current_stat)
+        if current_stat != value:
+            cmd = "sed -E -i 's/(^SELINUX=).*?/\\1%s/' /etc/selinux/config" % value
+            logging.info('Set selinux stat with command %s', cmd)
+            session.cmd(cmd)
+            return True
+        else:
+            return False
```
Comment (will-Do) [inline v2v/tests/src/specific_kvm.py]: It's weird to return False here

**Comment by chloerh** [comment]
@will-Do updated, please review, thanks.

File: v2v/tests/cfg/specific_kvm.cfg [hunk 3]
```diff
@@ -123,6 +121,8 @@
                     checkpoint = 'fstab_uuid'
                 - virtio:
                     checkpoint = 'fstab_virtio'
+                - wrong:
+                    checkpoint = 'fstab_wrong'
```
Comment (will-Do) [inline v2v/tests/cfg/specific_kvm.cfg]: So this should be a negative test case, right? How about name it to 'invalid'

**Comment by will-Do** [comment]
ACK

**Pull request merged**